webapp/README.md

```diff
@@ -1,6 +1,6 @@
 # Connected Lights Web Application
 
-This is the web application that goes with the article [Building an internet connected lighting system](), in which we build a lighting system on top of the ARM mbed IoT Device Platform.
+This is the web application that goes with the article [Building an internet connected lighting system](), in which we build a lighting system on top of the Pelion IoT Platform.
```

Comment:
This should highlight both Pelion and Mbed in this sentence. Maybe 'in which we build a lighting system on top of the Pelion IoT Platform and Mbed OS'.

Reply:
Got it